tools/pkg/engine_build_configs/lib/src/build_config_runner.dart

```diff
@@ -579,6 +579,27 @@ final class BuildRunner extends Runner {
     return min(multiplier * cores, 1000);
   }();
 
+  static final RegExp _gccRegex =
+      RegExp(r'^(.+)(:\d+:\d+:\s+(?:error|note|warning):\s+.*)$');
+
+  /// Converts relative [path], who is relative to [dirPath] to a relative path
+  /// of the `CWD`.
+  static String _makeRelative(String path, String dirPath) {
```

Comment:
I'm curious what this does meaningfully diferent than using the optional parameter `from` in `p.relative`: https://pub.dev/documentation/path/latest/path/relative.html?

Reply:
`from` is confusing, it replaces the CWD part of the equation not were the path is relative to.

Reply:
Ack, thanks!